tools/gulp/constants.ts

```diff
@@ -29,6 +29,12 @@ export const HTML_MINIFIER_OPTIONS = {
   removeAttributeQuotes: false
 };
 
+/** Packages that will be published to NPM by the release task. */
+export const RELEASE_PACKAGES = [
+  'cdk',
+  'material',
+];
```

Comment:
Yeah sorry if it was confusing. Let's also try to get rid of all the path constants in a follow-up then?

Because I don't think it makes sense to store constants like `DIST_MATERIAL` in a config?